Gaijinizer patch/Map216.txt

```diff
@@ -696,10 +720,10 @@ unquestionably and honestly#MTLed
  明らかに足元だったね……
 不意は突いてたけど、アレは当てる気がないって言うか…
 > CONTEXT: Map216/events/5/pages/1/290/Dialogue/kazuya
-If you try to say, bullets shot to reveal your
-true identity, Apparently it was my foot ......
-I had a surprise but did not say that I didn't
-want to touch it ...#MTLed
+Now that you mention it, the bullet I fired at
+you when you revealed yourself was aimed at your
+feet. It happened so fast but I didn't actually
+intend it to hit you...#RonIffy
 > END STRING
```

Comment:
just a bit of reworking:
_**Now that you mention it, the bullet I fired when
you revealed yourself was aimed at your feet.
It happened so fast, but I didn't actually intend
it to hit you...**_